### PR TITLE
FLW-1133 Replace Subscan API to SubQuery API in fetching Pending rewards flow

### PR DESCRIPTION
**Pull request by ef1rspb**

### CODE REVIEW

File: fearless/Common/Services/PayoutRewardsService/PayoutValidatorsForNominatorFactory.swift [hunk 1]
```diff
@@ -5,225 +5,156 @@ import IrohaCrypto
...
-            return [wrapper]
-        }.longrunOperation()
-
-        validatorsOperation.addDependency(controllersQueryWrapper.targetOperation)
-
-        let mapOperation = ClosureOperation<[AccountId]> {
-            try validatorsOperation.extractNoCancellableResultData().flatMap { $0 }
+    private func requestParams(accountAddress: AccountAddress, erasRange: [EraIndex]?) -> String {
+        let eraFilter: String = {
+            guard let range = erasRange, range.count >= 2 else { return "" }
```
Comment (ERussel) [inline fearless/Common/Services/PayoutRewardsService/PayoutValidatorsForNominatorFactory.swift:134]: I think here we have inclusive range. So, we can just say: `let fistRange = range.first, let lastRange = range.last` in the guard and then use them later without unwrapping?

File: fearless/Modules/Staking/StakingRewardPayouts/StakingRewardPayoutsViewFactory.swift [hunk 2]
```diff
@@ -25,9 +24,7 @@ final class StakingRewardPayoutsViewFactory: StakingRewardPayoutsViewFactoryProt
 
         let validatorsResolutionFactory = PayoutValidatorsForNominatorFactory(
             chain: chain,
-            subscanBaseURL: subscanUrl,
-            subscanOperationFactory: SubscanOperationFactory(),
-            operationManager: operationManager
+            subqueryURL: URL(string: "http://localhost:3000/")! // TODO: delete stub url
```
Comment (ERussel) [inline fearless/Modules/Staking/StakingRewardPayouts/StakingRewardPayoutsViewFactory.swift:27]: Don't forget to fix this

File: fearless/Common/Services/PayoutRewardsService/PayoutValidatorsForNominatorFactory.swift [hunk 3]
```diff
@@ -5,225 +5,156 @@ import IrohaCrypto
...
-                    address: address,
-                    moduleName: "utility",
-                    callName: "batch_all",
-                    mapper: AnyMapper(mapper: BatchMapper(innerMapper: mapper)),
-                    reducer: AnyReducer(reducer: NominationsListReducer())
-                )
-
-                return [validatorsByNominate, validatorsByBatch, validatorsByBatchAll]
-            }
+// TODO: move to /Common/DataProvider/Subquery when Analytics will be done
```
Comment (ERussel) [inline fearless/Common/Services/PayoutRewardsService/PayoutValidatorsForNominatorFactory.swift:73]: Don't forget to move this

File: fearless/Common/Services/PayoutRewardsService/PayoutValidatorsForNominatorFactory.swift [hunk 4]
```diff
@@ -5,225 +5,156 @@ import IrohaCrypto
...
-        }
-
-        let dependencies = [bondOperation, setControllerOperation]
-        dependencies.forEach { mergeOperation.addDependency($0) }
-
-        return CompoundOperationWrapper(
-            targetOperation: mergeOperation,
-            dependencies: dependencies
-        )
+// TODO: move to /Common/Network/Subquery when Analytics will be done
```
Comment (ERussel) [inline fearless/Common/Services/PayoutRewardsService/PayoutValidatorsForNominatorFactory.swift:34]: Don't forget to move this

File: fearless/Common/Services/PayoutRewardsService/PayoutValidatorsForNominatorFactory.swift [hunk 5]
```diff
@@ -5,225 +5,156 @@ import IrohaCrypto
...
+            guard
+                let resultData = try? JSONDecoder().decode(JSON.self, from: data),
+                let nodes = resultData.data?.query?.eraValidatorInfos?.nodes?.arrayValue
+            else { return [] }
+
+            let addressFactory = SS58AddressFactory()
+            let validators = nodes
+                .compactMap { SQEraValidatorInfo(from: $0) }
+                .compactMap { validatorInfo -> AccountAddress? in
+                    let contains = validatorInfo.others.contains(where: { $0.who == self.address })
```
Comment (ERussel) [inline fearless/Common/Services/PayoutRewardsService/PayoutValidatorsForNominatorFactory.swift:121]: Why do we need this filter here?

Comment (ef1rspb) [reply fearless/Common/Services/PayoutRewardsService/PayoutValidatorsForNominatorFactory.swift:121]: Subquery returns array of items as string where could be others.who != queryAddress

**Pull request closed without merging**